main.js

```diff
@@ -65,6 +65,11 @@ async function downloadCLI(version, platform, arch) {
     core.info(`⬇️ Downloading UpCloud CLI from ${downloadUrl}`);
     
     const downloadPath = await tc.downloadTool(downloadUrl);
+    if (false) { // TODO if >= v3.16.0
```

Comment:
Do we want this action to work for downloading older versions as well, or could we draw a line and state that upctl older than v3.16.0 can not be installed with this version of the action?

Reply:
Would `verify-attestation: false` allow downloading versions < 3.16 as well? I guess most users would want more recent version anyway.

Reply:
I think it would.